src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
+import Navbar from "../navbar";
+import EnvironmentBar from "./environmentBar";
+
+const useStyles = makeStyles((theme) => ({
+    page: {
+        marginTop: theme.spacing(8),
+    },
+}));
+
+export default function EnvironmentEditor() {
+    const classes = useStyles();
+
+    return (
+        <div>
+            <Navbar home />
+            <div className={classes.page}>
+                <EnvironmentBar />
+                Hello
```

Comment:
Can remove this line, isn't even showing in the loom video (probably needs some padding top).

Alternatively, can do a <div> below environment bar with a paddingTop that will handle the navbar + env bar heights.

Reply:
Ah forgot about this, deleted.